test/unit/reducers/projects.js

```diff
@@ -164,6 +172,30 @@ tap(initProjects({1: false}), projects =>
   )),
 );
 
+tap(initProjects({1: false}), projects =>
+  test('hideComponent', reducerTest(
+    reducer,
+    projects,
+    partial(hideComponent, '1', 'output', now),
+    projects.update('1', projectIn =>
+      projectIn.set('hiddenUIComponents', new Immutable.Set(['output'])).
+        set('updatedAt', now),
+    ),
+  )),
+);
+
+tap(initProjects({1: true}), projects =>
+  test('unhideComponent', reducerTest(
+    reducer,
+    projects.update('1', projectIn =>
+      projectIn.set('hiddenUIComponents', new Immutable.Set(['output'])).
+        set('updatedAt', now),
```

Comment:
Yep, agreed that omitting the `updatedAt` is probably best since it removes a peripheral concern from the test, although I also don’t feel terribly strong about it.